test/parallel/test-crypto-authenticated.js

```diff
@@ -68,9 +68,8 @@ const TEST_CASES = [
     ct: 'dda53a4059aa17b88756984995f7bba3c636cc44',
     tag: 'd2a35e5c611e5e3d2258360241c5b045', tampered: false },
 
-  // Following test cases are from
-  //   http://csrc.nist.gov/groups/ST/toolkit/BCM/documents/
-  //    proposedmodes/gcm/gcm-revised-spec.pdf
+  // Following test cases are from "The Galois/Counter Mode of Operation (GCM)"
+  // by D. McGrew and J. Viega, published by the Natl. Inst. Stand. Technol.
```

Comment:
`National Institute of Standards and Technology`?

Reply:
Yeah... `Natl. Inst. Stand. Technol.` is the abbreviation which is used in their own document, I just copied it. Personally, I would just write NIST, what do you prefer?

Reply:
NIST LGTM.

Reply:
I think NIST would be better.